packages/headless/src/controllers/core/facets/category-facet/headless-core-category-facet-options.ts

```diff
@@ -110,6 +115,15 @@ export const categoryFacetOptionsSchema = new Schema<
   Required<CategoryFacetOptions>
 >({
   field,
+  tabs: new RecordValue({
+    options: {
+      required: false,
+    },
+    values: {
+      included: new ArrayValue({each: new StringValue()}),
```

Comment:
Also, should there be some kind of validation that someone is specifying a tab that actually exist ?
Example: 
An interface has `tab1`, `tab2`.
Someone configure a facet to be included in `tab3` (which does not exist)

That's an illogical configuration since I think (?) it means the facet will never be shown/enabled for the end user, so it's useless.

Flip side, we could also say that implementer could do something like this and that it would then be logical: register a new facet on `tab3` and then register a new tab called `tab3` later in the page execution.

Reply:
I think tabs that don't exist should be handled when changing the active tab instead of when registering a tab, but I'm interested in hearing what the rest of the team thinks